cfme/tests/pod/test_appliance_crud.py

```diff
@@ -324,3 +342,59 @@ def test_aws_smartstate_pod():
         initialEstimate: None
     """
     pass
+
+
+def test_pod_appliance_db_backup_restore(temp_pod_appliance, provider, setup_provider,
+                                         template_folder):
+    """
+    This test does the following:
```

Comment:
Polarion metadata please, at least assignee and initial estimate

Reply:
it is already done in next PR